Drop the include of libspeechd_version.h

libspeechd_version.h is generated by upstream's build
(libspeechd_version.h.in) and is not imported: Chromium only needs the
declarations to load libspeechd.so.2 at runtime (README.chromium).

diff --git a/src/libspeechd.h b/src/libspeechd.h
index 2b950857b3269..6c08434a9d6e9 100644
--- a/src/libspeechd.h
+++ b/src/libspeechd.h
@@ -26,7 +26,6 @@
 #include <stddef.h>
 #include <pthread.h>
 
-#include "libspeechd_version.h"
 #include "speechd_types.h"
 
 /* *INDENT-OFF* */
